icons/x.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.233 10.157 22.971 0h-2.07l-7.588 8.82L7.253 0H.264l9.164 13.336L.264 23.988h2.07l8.013-9.314 6.4 9.314h6.989l-9.503-13.83Zm-2.836 3.297-.928-1.328L3.08 1.559h3.18l5.962 8.528.929 1.328 7.75 11.085H17.72l-6.324-9.046Z"/></svg>
```

Comment:
You need to use `viewBox` instead of width and height so that the icon is responsive. Also, can you remove `fill="none"` so that we can see by default what the icon look like please ?

Reply:
also, I just observed that `youtube.svg` is invalid because missing `xmlns="http://www.w3.org/2000/svg"`, ca you add it please ?